### PR TITLE
Ankura2

### PR DESCRIPTION
**Pull request by wfearn**
bitmoji

### CODE REVIEW

File: ankura/anchor.py [hunk 1]
```diff
@@ -157,7 +157,7 @@ def build_supervised_cooccurrence(corpus, attr_name, labeled_docs):
                         continue
                     S[w_i.token, index] += 1
     for i in range(S.shape[0]):
-        S[i,:] /= np.sum(S[i,:])
+        S[i,:] = (S[i,:] / np.sum(S[i,:]))
```
Comment (jefflund) [inline ankura/anchor.py]: bitmoji

**Review by jefflund — CHANGES_REQUESTED** [review]

**Comment by wfearn** [comment]
bitmoji

**Review by jefflund — APPROVED** [review]
https://render.bitstrips.com/v2/cpanel/2e305829-118b-46b5-953a-bc8b7ff51779-58e1cf7a-21f2-4c43-82cb-72edb2944d01-v1.png?transparent=1&palette=1

**Comment by jefflund** [comment]
bitmoji

**Pull request merged**